monorail/world.py

```diff
@@ -31,7 +32,8 @@ def set_tile( self, tile ):
         self.remove_tile( tile.pos.x, tile.pos.y )
         self.tiles.append( tile )
 
-        self.tiles.sort( tilesort )
+        # TODO: check for reverse
```

Comment:
In my opinion, it is better to create an issue. There is no contribution guide to say something about `TODO` convention/workflow, this means everyone will think what appeals to them.